src/Configurations/test/setup.js

```diff
@@ -0,0 +1,35 @@
+import { BrowserRouter } from 'react-router-dom';
+import { IntlProvider } from 'react-intl';
+
+import { render } from '@testing-library/react';
+
+import { CONFIGURATIONS_PATH } from '../../const';
+import { Configurations } from '../Configurations';
+import { API_BASE, Provider } from '../../test/net';
+
+export const renderConfigurations = route => {
+  const path = [CONFIGURATIONS_PATH, route].join('');
+
+  window.history.pushState({}, 'Test page', path);
+
+  return render(
+    (
+      <BrowserRouter>
+        <IntlProvider locale="en">
+          <Configurations />
+        </IntlProvider>
+      </BrowserRouter>
+    ),
+    { wrapper: Provider },
+  );
+};
+
+export const url = {
+  providers: `${API_BASE}/providers`,
+  configurations: {
+    single: `${API_BASE}/configurations/1`,
```

Comment:
"1" here is an ID from the fixture, which is now in another file.
I don't have a ready solution how to organize this things for now.
Got to think it over...